Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java

```diff
@@ -770,6 +770,30 @@ public String deleteTask() {
 
     }
 
+    // TODO deleteTask() and removeTask() can be merged, when old templates have been removed
+    /**
+     * Remove task.
+     *
+     */
+    public void removeTask() {
+        try {
+            this.process.getTasks().remove(this.task);
+            List<User> users = this.task.getUsers();
+            for (User user : users) {
+                user.getTasks().remove(this.task);
+            }
+
+            List<UserGroup> userGroups = this.task.getUserGroups();
+            for (UserGroup userGroup : userGroups) {
+                userGroup.getTasks().remove(this.task);
+            }
+            deleteSymlinksFromUserHomes();
+            serviceManager.getTaskService().remove(this.task);
+        } catch (DataException e) {
+            logger.error(e);
```

Comment:
In a previous pull request is a new Helper method added which logs the error and inform the user in the UI. Would this not a better approach?